src/lib/eina/eina_inline_lock_win32.x

```diff
@@ -524,90 +441,35 @@ _eina_tls_get(Eina_TLS key)
 static inline Eina_Bool
 _eina_tls_set(Eina_TLS key, const void *data)
 {
-   int ok = TlsSetValue(key, (void *) data);
-   DWORD err = GetLastError();
-   if (ok != 0 && err == ERROR_SUCCESS) return EINA_TRUE;
-   else return EINA_TRUE;
+   return TlsSetValue(key, (void *) data) ? EINA_TRUE : EINA_FALSE;
 }
 
-#ifdef EINA_HAVE_WIN32_BARRIER
```

Comment:
If we are removing this, we can remove its definition at `src/lib/eina/meson.build`

Reply:
You are right!